src/components/Card/CardToolbar.jsx

```diff
@@ -48,9 +51,24 @@ const CardToolbar = ({
   isExpanded,
   availableActions,
   timeRange,
+  timeRangeOptions,
   onCardAction,
   className,
 }) => {
+  // maps the timebox internal label to a translated string
+  // Need the default here in case that the CardToolbar is used by multiple different components
+  const timeBoxLabels = {
+    last24Hours: i18n.last24HoursLabel,
+    last7Days: i18n.last7DaysLabel,
+    lastMonth: i18n.lastMonthLabel,
+    lastQuarter: i18n.lastQuarterLabel,
+    lastYear: i18n.lastYearLabel,
+    thisWeek: i18n.thisWeekLabel,
+    thisMonth: i18n.thisMonthLabel,
+    thisQuarter: i18n.thisQuarterLabel,
+    thisYear: i18n.thisYearLabel,
+  };
```

Comment:
Should this be in a useEffect so it's updated if the i18n props change?

Reply:
Since its not in a hook, this should get re-called with any re-render, including a prop change. I could see the benefit of memoizing it maybe?

Reply:
Or I should set these as the default i18n in CardRangePicker

Reply:
Went ahead and memoized the defaults based off the i18n and timeRangeOptions prop